frontend/src/pages/geography.tsx

```diff
@@ -0,0 +1,825 @@
+      });
+
+      // Add a layer showing the countries polygons.
+      map.addLayer({
+        id: "countries-layer",
+        type: "fill",
+        source: "countries",
+        paint: {
+          "fill-color": [
+            "match",
+            ["get", "ISO_A3"],
+            countriesWithLargeAmountOfDatasets,
+            "#006064",
+            countriesWithMediumAmountOfDatasets,
+            "#00BCD4",
+            "#D1D5DB",
+          ],
+          "fill-outline-color": "white",
+        },
+      });
+
+      map.on("click", "countries-layer", (e) => {
+        router.push(
+          `/dataset-search?country=${(e.features || [])[0]?.properties.ISO_A2}`
+        );
+      });
+
+      map.on("mouseenter", "countries-layer", () => {
+        map.getCanvas().style.cursor = "pointer";
+      });
+
+      map.on("mousemove", "countries-layer", (e) => {
+        if (popup._container) {
+          popup._container.style.minWidth = "194px";
+          popup._container.style.cursor = "pointer";
+          popup._container.style.width = "194px";
+          popup._container.style.height = "171px";
+          popup._container.style.minHeight = "171px";
+        }
+
+        if (
+          (e.features || []).length > 0 &&
+          !(
+            (e.features || [])[0]?.properties.ADMIN ===
+            popup
+              .getElement()
+              ?.getElementsByTagName("div")
+              ?.item(1)
+              ?.getElementsByTagName("div")
+              ?.item(0)
+              ?.getElementsByClassName("country-title")
+              ?.item(0)
+              ?.textContent?.trim()
+          )
+        ) {
+          popup
+            .setLngLat(e.lngLat)
+            .setHTML(
+              `        
+              <div>
+              <img style="width: 40px; height: 40px; border-radius: 9999px" src="https://flagsapi.com/${
+                (e.features || [])[0]?.properties.ISO_A2
+              }/flat/64.png"></img>
+                
+              <div class="country-title" style="color: white'; font-size: 14px">${
+                (e.features || [])[0]?.properties.ADMIN
+              }
+              </div>
+              <div style="color: #9CA3AF">${
+                (e.features || [])[0]?.properties.ISO_A2
+              }</div>
+
+
+                </div>
+                
+                <div style="color: white; font-size: 30px">${1650}+</div>
+                <div style="color: #9CA3AF; font-size: 16px">Datasets</div>
+              `
+            )
+            .addTo(map);
+        }
+      });
+
+      map.on("mouseleave", "countries-layer", () => {
+        map.getCanvas().style.cursor = "";
+        popup.remove();
+      });
+    });
+  }, []);
+
+  return (
+    <>
+      <Head>
+        <title>Geography</title>
+        <meta name="description" content="Generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Layout backgroundEffect>
```

Comment:
Try playing with the backgroundEffect (There is a prop to change the percentage) curreently you can see a slight difference in the color 
![image](https://github.com/user-attachments/assets/3338cd40-e5f4-408b-a535-55586f00f1ef)